pkg/koyeb/koyeb.go

```diff
@@ -61,6 +61,21 @@ func genericArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func isHelpCalled(cmd *cobra.Command) bool {
+	for _, subcmd := range cmd.Commands() {
+		if subcmd.Name() == "help" {
+			return subcmd.CalledAs() != ""
+		}
+	}
+
+	return false
+}
```

Comment:
You might wonder why checking if help is called is so strange.

Unfortunately it seems that there is no getter for [helpCommand](https://github.com/spf13/cobra/blob/284f4101043c4b1faebde411caec35a7b6e36494/command.go#L174) so the only way is to check if help was called is to:
1. list all subcommands of root cmd
2. find help cmd in it
3. check if help cmd was called with [CalledAs](https://pkg.go.dev/github.com/spf13/cobra#Command.CalledAs)